Python Tutorial Exercises(Week 5)/#4.py

```diff
@@ -0,0 +1,26 @@
+'''
+use if len(i) >= 8 to find all the strings that have at least 8 characters or more.
+    characters = [char for char in andrews_passwords if len(char) >= 8]
+
+use if not sym.isalnum() to get the strings with least one character is not alphanumeric
+    symbol = [sym for sym in andrews_passwords if not sym.isalnum()]
+
+use if sum(map(str.islower, low)) >= 1 to get the strings with least one character is lowercase letter
+    lowercase  = [low for low in andrews_passwords if sum(map(str.islower, low)) >= 1]
```

Comment:
Have a comment regarding what the `map()` function does, and is it necessary in this scenario